### PR TITLE
Dev --> master

### PR DESCRIPTION
**Pull request by gemfarmer**
[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/dev)

[:sunglasses: PREVIEW](https://federalist.18f.gov/preview/18F/18f.gsa.gov/dev/)

Changes proposed in this pull request:
- adds `./serve-blog` script
- adds new hhs project page

/cc @coreycaitlin

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,10 +1,3 @@
-GIT
```
Comment (gemfarmer) [inline Gemfile.lock]: These changes need to be reverted before merging

**Comment by gemfarmer** [comment]
LGTM! !!

**Comment by coreycaitlin** [comment]
@gboone This look okay to merge?

**Comment by gemfarmer** [comment]
Is it time to press the big 🍏  button?

**Pull request merged**